templates/OperationSnippet.stencil

```diff
@@ -1,4 +1,11 @@
 {{ op.comment }}
+/// - Parameters:
+{% for param in op.params %}
+///   - {{ param.name }}: {{ param.comment }}
+{% endfor %}
+///   - options: A list of options for the operation
+///   - completionHandler: A completion handler that receives a status code on
+///     success.
```

Comment:
The only problem here is that we will have lines generated even where there is no doc-comment. It might make sense to build the entire comment up in `ViewModelComment`.

Reply:
Good point.